frontend/src/styles/theme.ts

```diff
@@ -6,6 +6,7 @@ const color = {
   PRIMARY200: '#b7baff',
   PRIMARY900: '#5054ff',
   WHITE: '#fff',
+  WHITE_BLUR: 'rgba(255, 255, 255, 0.68)',
```

Comment:
요거만 일관성에 맞게 HEX 값으로 바꿔주면 좋을듯!
```suggestion
  WHITE_BLUR: '#ffffffad',
```